CHANGES.md

```diff
@@ -209,7 +212,6 @@ Deprecations:
 [David Ostrovsky]: https://github.com/davido
 [AndyKIron]: https://github.com/AndyKIron
 [Samuel Colvin]: https://github.com/samuelcolvin
-[Jan Pilzer]: https://github.com/Hirse
```

Comment:
Why?

Reply:
I didn't have a contribution in 10.7, but did have one in 10.6 so just moved this line down one block.

Reply:
Ah. ok :) Oops.